pkg/util/priority/priority.go

```diff
@@ -75,13 +75,20 @@ func getDefaultPriorityClass(ctx context.Context, client client.Client) (*schedu
 
 	// In case more than one global default priority class is added as a result of a race condition,
 	// we pick the one with the lowest priority value.
-	var defaultPC *schedulingv1.PriorityClass
+	var defaultPC schedulingv1.PriorityClass
```

Comment:
Nit: if you are still returning a pointer, I think the initial implementation was simpler, as it does not need a boolean.